live-build/variants/internal-hyperscale/ansible/playbook.yml

```diff
@@ -24,9 +24,9 @@
     - appliance-build.minimal-common
     - appliance-build.minimal-internal
     - appliance-build.minimal-development
-    - appliance-build.hyperscale-common
     - appliance-build.masking-common
     - appliance-build.masking-development
     - appliance-build.virtualization-common
     - appliance-build.virtualization-development
+    - appliance-build.hyperscale-common
```

Comment:
Can we please add a comment, that `hyperscale-common` is dependent on `masking-common` and requires to install `masking-common` first before installing `hyperscale-common`